src/viam/sdk/services/mlmodel/private/proto.hpp

```diff
@@ -22,13 +22,8 @@ namespace viam {
 namespace sdk {
 namespace mlmodel_details {
 
-template <typename T>
-struct make_vector {
-    using type = std::vector<T>;
-};
-
 using tensor_storage_types = boost::mpl::transform_view<MLModelService::base_types,
-                                                        make_vector<boost::mpl::placeholders::_1>>;
+                                                        std::vector<boost::mpl::placeholders::_1>>;
```

Comment:
Figured out that `make_vector` wasn't needed at all, `mpl` can do the right thing automatically.